py/modsys.c

```diff
@@ -60,15 +60,24 @@ STATIC const MP_DEFINE_STR_OBJ(mp_sys_version_obj, "3.4.0; " MICROPY_BANNER_NAME
 
 // version_info - Python language version that this implementation conforms to, as a tuple of ints
 #define I(n) MP_OBJ_NEW_SMALL_INT(n)
-// TODO: CPython is now at 5-element array, but save 2 els so far...
+// TODO: CPython is now at 5-element array (major, minor, micro, releaselevel, serial), but save 2 els so far...
```

Comment:
Comment needs updating "but save 1 element so far"

Reply:
It's still missing the last two. This is the Python version (i.e. `3.4.0`) -- I didn't change the tuple contents, only the comment.

Reply:
Hm, I see.  I'm confused because in CPython `sys.version_info` and `sys.implementation.version` are exactly the same object.  But obviously not for us....